README.md

```diff
@@ -9,11 +9,14 @@ The following environment variables can be set:
 
 Name | description | default 
 --- | --- | --- 
+DOCKER_CERTIFICATE_SECRET | You **must** provide a certificate to enable TLS between the docker daemon and the registry | None
```

Comment:
is this the name of a secret in the same namespace or the base64 blob of secret contents?

Reply:
I have amended the docs to say this is the name of the secret